README.md

```diff
@@ -1,11 +1,34 @@
-# Automatically deploy Lightdash projects
+# Lightdash GitHub Actions templates
 
-Follow the [lightdash documentation](#TODO) to know how you can add this Github action to your DBT Github repo to automatically deploy your DBT changes into Lightdash
+## Refresh your Lightdash project
+> refresh.yml
 
+Follow the [Lightdash documentation](https://docs.lightdash.com/references/syncing_your_dbt_changes) to know how you can
```

Comment:
This doc is a bit confusing now that we have different actions, on `Step 2: Create action.yml workflow in Github` it points to this file https://github.com/lightdash/cli-actions/blob/main/action.yml

Reply:
I can:
- change this url to `https://docs.lightdash.com/references/syncing_your_dbt_changes/#3-automatically-deploy-your-changes-to-lightdash-using-a-github-action`
- update the docs link to the template since I renamed it to `deploy.yml` 
In the later future:
- update docs to explain the `refresh` command and github template https://github.com/lightdash/lightdash/issues/3629

Reply:
Forget what I said, I'm going to leave this section blank since we haven't written the docs for this template.